### ISSUE TITLE
Github Pages Site?

### ISSUE THREAD

**Issue by reignrunner** [y2s82/goblin_camp#130]
We may want to add a Github pages website to this repo, I would gladly assist in setting that up.

**Comment by gabriel-francischini** [comment]
Agreed. Heck, with things like Emscripten, WebAssembly and SDL we may even be able to have a version that is able to run in the browser! (possibly in github pages?)
